core/src/main/java/io/parsingdata/metal/Shorthand.java

```diff
@@ -136,10 +135,6 @@ private Shorthand() {}
     public static Token opt(final String name, final Token token) { return opt(name, token, null); }
     public static Token opt(final Token token, final Encoding encoding) { return opt(NO_NAME, token, encoding); }
     public static Token opt(final Token token) { return opt(token, null); }
-    public static Token nod(final String name, final ValueExpression size) { return new Nod(name, size, null); }
```

Comment:
I would opt to keep the nod as shorthand. Else you are obligated to always specify a name for unknown data, which will probably result in unknown1, unknown2, etc.

Reply:
I agree. In practice I think there is a valid use case for `nod()`.

Reply:
I will reinstate two of the four shorthands: only the ones without a name argument (because if you want to specify a name, use `def`).